### PR TITLE
Type issue

### PR DESCRIPTION
**Pull request by ScHcSII**
Hi

Attempt to fix the issue:
AttributeError with Icinga DB 1.2.0 / Icinga DB Web 1.1.2 #1028 Nagstamon Error: AttributeError: 'bool' object has no attribute 'replace'

First pull request ever so please review carefully.
Slightly modified two lines, my nagstamon works after the changes.

### CODE REVIEW

File: Nagstamon/Servers/IcingaDBWeb.py [hunk 1]
```diff
@@ -220,7 +220,8 @@ def _get_status(self):
                         self.new_hosts[host_name].notifications_disabled = not int(h.get('notifications_enabled') or '0')
                         self.new_hosts[host_name].flapping = bool(int(h['state']['is_flapping'] or 0))
                         #s['state']['is_acknowledged'] can be null, 0, 1, or 'sticky'
-                        self.new_hosts[host_name].acknowledged = bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        #self.new_hosts[host_name].acknowledged = bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        self.new_hosts[host_name].acknowledged = h['state']['is_acknowledged'] if isinstance(h['state']['is_acknowledged'], (bool)) else bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
```
Comment (HenriWahl) [inline Nagstamon/Servers/IcingaDBWeb.py:224]: Shouldn't it work with `bool` instead of `(bool)` too?

File: Nagstamon/Servers/IcingaDBWeb.py [hunk 2]
```diff
@@ -323,7 +324,8 @@ def _get_status(self):
                         self.new_hosts[host_name].services[service_name].notifications_disabled = not int(s.get('notifications_enabled') or '0')
                         self.new_hosts[host_name].services[service_name].flapping = bool(int(s['state']['is_flapping'] or 0))
                         #s['state']['is_acknowledged'] can be null, 0, 1, or 'sticky'
-                        self.new_hosts[host_name].services[service_name].acknowledged = bool(int(s['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        #self.new_hosts[host_name].services[service_name].acknowledged = bool(int(s['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        self.new_hosts[host_name].services[service_name].acknowledged = s['state']['is_acknowledged'] if isinstance(s['state']['is_acknowledged'], (bool)) else bool(int(s['state']['is_acknowledged'].replace('sticky', '1') or 0))
```
Comment (HenriWahl) [inline Nagstamon/Servers/IcingaDBWeb.py:328]: Same here. :smile:

File: Nagstamon/Servers/IcingaDBWeb.py (hunk 1 above)
Comment (magenbrot) [reply Nagstamon/Servers/IcingaDBWeb.py:224]: works for me with the correction from @HenriWahl

File: Nagstamon/Servers/IcingaDBWeb.py (hunk 2 above)
Comment (magenbrot) [reply Nagstamon/Servers/IcingaDBWeb.py:328]: Same here. 😄

**Comment by HenriWahl** [comment]
Please check latest release 3.15-20240503.

**Pull request merged**